aloscene/tensors/augmented_tensor.py

```diff
@@ -699,6 +699,10 @@ def as_tensor(self):
         n_tensor.__class__ = torch.Tensor
         n_tensor.rename_(None)
         return n_tensor
+    
+    def to_squeezed_numpy(self):
```

Comment:
When we think about it, squeeze could be application specific and/or could be made after in numpy
```
np.squeeze(tensor.numpy(), axis=0)
tensor.numpy()[0] # or like this
```
I think this method could be usefull to make it quicker to export to numpy without having to detach & pass to the cpu each time. 

Also the type might be something we want to be set as parameter. 

Finally, its best to first detach, then pass to the cpu (otherwise you copy all the computation graph to the cpu). 

Here is my proposition
- We already have a method named `as_tensor`. Following the same format this method could be named `as_numpy`
- This is different from `tensor.numpy()` that assume you're already on the cpu & detach from graph.
- `as_numpy` will detach, move the tensor on the cpu & finally convert to the expected type.

```
tensor.as_numpy()
tensor.as_numpy(np.float16)
```

Let me know if it make sense to you or if you have an other proposition.